tests/pythonSimulator/README.md

```diff
@@ -0,0 +1,288 @@
+
+# OpenOffload Python Simulator
+
+gRPC API to offload TCP and UDP stateful packet processing from an application to a hardware device.  The simulator consists of two main scripts.  The first script we execute is the nos_sessions_server_simulator.py.  This script should be thought of as the hardware device that implements the stateful session offload and implements the gRPC server side. The second script we execute is the fw_sessions_client_simulator.py. This second script should be thought of as the firewall device which desires to offload stateful existing sessions.
```

Comment:
is there a way to create entry points that robot could call to run a specific test (multiple adds for instance) and then get a SUCCESS/FAILURE on the expected behavoir or a robot generated addSession etc sequence that coulid start from a  fresh server side and get SUCCESS/FAILURE at the end of the sequence ?

Reply:
Yes I'll add cmdline options next to the commands.

Reply:
I will add cmdline options to the simulators so that we can unit test.  For example: --addSession JSON payload   and --getSession ...  This way we can unit test through the simulator.